assets/images/sprite/logo.svg

```diff
@@ -0,0 +1,4 @@
+</svg>
```

Comment:
Why is there a background element within this SVG image; shouldn't this be an external HTML element that wraps the SVG element?

Reply:
It is a basic example, just in case an icon has a background (or any child really) with a different color (rectangle, square, circle, ellipsis, dot ...)